package.json

```diff
@@ -39,27 +39,28 @@
     "check": "npm-check -i app -i errors -i config -i index & exit 0"
   },
   "dependencies": {
-    "@types/express": "^4.16.0",
-    "@types/express-serve-static-core": "^4.16.1",
+    "@types/express": "^4.17.8",
+    "@types/express-serve-static-core": "^4.17.13",
+    "@types/lodash": "^4.14.162",
```

Comment:
:question: do we need lodash types?

Reply:
Omit types was used, just the dep was missing. I could rewrite the types myself, if you prefer.